staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -78,13 +82,16 @@ func (o *Options) Get() (logr.Logger, error) {
 	return logRegistry.Get(o.LogFormat)
 }
 
-func unsupportedLoggingFlags() []string {
+// unsupportedLoggingFlags validates all the flags provided
+// if any unsupported falg is provided
+// returns true, with list of all unsupported flags
+func unsupportedLoggingFlags() (bool, []string) {
```

Comment:
We should not return bool here, maybe just check if slice is empty on caller side?